scm/src/gmtb_scm_type_defs.f90

```diff
@@ -69,9 +70,30 @@ module gmtb_scm_type_defs
     logical                           :: sfc_flux_spec !< flag for using specified surface fluxes instead of calling a surface scheme
     integer                           :: sfc_type !< 0: sea surface, 1: land surface, 2: sea-ice surface
     real(kind=dp)                     :: sfc_type_real(1)
+    integer                           :: veg_type !
```

Comment:
Note to self: it may be advantageous to move these variables to a different type. The "unwritten" rule that I had been following with respect to what belongs in this data type is that it should contain variables that are either static and represent some geophysical property or that they are subject to forcing. Many of these could simply be read in to the input DDT and simply used to initialize corresponding variables in one of the other GFS DDTs (or simply variables in the physics DDT if GFS DDTs are eliminated for the SCM in the future).

Reply:
Also, consider whether these should be array size (1) or allocatable for n_columns.